.github/workflows/main.yml

```diff
@@ -332,6 +316,40 @@ jobs:
       env:
         RUST_BACKTRACE: 1
 
+  # Build and test the C interface
+  test_cmake:
+    name: Test C Interface
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+          - os: windows-latest
```

Comment:
I know windows is weird but we're tight enough on CI budget I'd be ok to drop windows here and just test on Linux